src/components/image/image.js

```diff
@@ -78,5 +78,5 @@ Image.defaultProps = {
   rounded: false,
   style: undefined,
   size: undefined,
-  fallback: 'http//bulma.io/images/placeholders/480x480.png',
+  fallback: 'http://bulma.io/images/placeholders/480x480.png',
```

Comment:
Thank you for the PR! Just one small fix: can you replace http with https? The http url will redirect to https.

Reply:
You're very welcome. Right thats a good idea.